flutter/Graminsta/lib/auth/signup.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:Graminsta/custom_icons_icons.dart';
+
+class SignupPage extends StatefulWidget {
+  @override
+  _SignupPageState createState() => _SignupPageState();
+}
+
+class _SignupPageState extends State<SignupPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(20),
+      child: Column(
+        children: <Widget>[
+          _signupForm(),
+          MaterialButton(
+            color: Colors.blue,
+            textColor: Colors.white,
+            child: Text('Register'),
+            onPressed: () {},
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget _signupForm() {
+    return Container(
+      width: 300,
+      height: 300,
+      child: Form(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            Flexible(
+              child: TextFormField(
+                decoration: InputDecoration(
+                  icon: Icon(
+                    Icons.account_box,
+                  ),
+                  hintText: "User Name",
+                ),
+                style: TextStyle(fontSize: 16, color: Colors.black),
+              ),
+            ),
+            Flexible(
+              child: Row(
+                  mainAxisSize: MainAxisSize.min,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    Expanded(
+                      child: TextFormField(
+                        decoration: InputDecoration(
+                          icon: Icon(
+                            Icons.account_box,
+                          ),
+                          hintText: "First Name",
+                        ),
+                        style: TextStyle(fontSize: 16, color: Colors.black),
+                      ),
+                    ),
+                    Expanded(
+                      child: TextFormField(
+                        decoration: InputDecoration(
+                          icon: Icon(
+                            Icons.account_box,
+                          ),
+                          hintText: "Last Name",
+                        ),
+                        style: TextStyle(fontSize: 16, color: Colors.black),
+                      ),
+                    ),
+                  ]),
+            ),
+            Flexible(
+              child: Row(
+                  mainAxisSize: MainAxisSize.min,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    Expanded(
+                      child: TextFormField(
+                        decoration: InputDecoration(
+                          icon: Icon(
+                            CustomIcons.gender,
+                          ),
+                          hintText: "Gender(F/M)",
+                        ),
+                        style: TextStyle(fontSize: 16, color: Colors.black),
+                      ),
```

Comment:
https://api.flutter.dev/flutter/material/RadioListTile-class.html?

Reply:
The logic of RadioListTile will be more complicated, and the UI will be more difficult to adjust. So I directly choose TextFormField.